### PR TITLE
Create new UI for the App

### PR DESCRIPTION
**Pull request by thisisaditya17**
Change up the UI based on this sketch 
#62

### CODE REVIEW

File: src/main/java/seedu/address/logic/PersonDataReceiver.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        this.overviewPanel = overviewPanel;
+    }
+
+    /**
+     * Processes the received person data.
+     *
+     * @param person The person whose data is being received.
+     */
+    public void receivePersonData(Person person) {
+        if (person != null) {
```
Comment (rithanisk) [inline src/main/java/seedu/address/logic/PersonDataReceiver.java:25]: Good job checking for null values!

File: src/main/java/seedu/address/ui/MainWindowNew.java [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+                event.consume();
+            }
+        });
+    }
+
+    void fillInnerParts() {
+
+        this.resultDisplay = new ResultDisplay();
+        resultDisplayPlaceholder.getChildren().add(resultDisplay.getRoot());
+
```
Comment (rithanisk) [inline src/main/java/seedu/address/ui/MainWindowNew.java:126]: Good use of spacing within method

**Comment by rithanisk** [comment]
LGTM!

File: src/main/java/seedu/address/ui/MainWindowNew.java [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+        this.overviewPanel = new OverviewPanel();
+        overviewPanelPlaceholder.getChildren().add(overviewPanel.getRoot());
+
+        this.personListPanel = new PersonListPanel(logic.getFilteredPersonList(), overviewPanel);
+        personListPanelPlaceholder.getChildren().add(personListPanel.getRoot());
+
+    }
+
+    private void setWindowDefaultSize(GuiSettings guiSettings) {
```
Comment (rithanisk) [inline src/main/java/seedu/address/ui/MainWindowNew.java:138]: Method adheres to SLAP to a reasonable extent - could it be abstracted even further?

Comment (rithanisk) [reply src/main/java/seedu/address/ui/MainWindowNew.java:138]: Does the method adhere to the Law of Demeter?

File: src/main/java/seedu/address/ui/MainWindowNew.java [hunk 4]
```diff
@@ -0,0 +1,220 @@
...
+        primaryStage.setHeight(guiSettings.getWindowHeight());
+        primaryStage.setWidth(guiSettings.getWindowWidth());
+        if (guiSettings.getWindowCoordinates() != null) {
+            primaryStage.setX(guiSettings.getWindowCoordinates().getX());
```
Comment (rithanisk) [inline src/main/java/seedu/address/ui/MainWindowNew.java:142]: Does this line adhere to the Law of Demeter?

**Review by dominic2412 — COMMENTED** [review]
Adi, this is very nice looking Ui! thank you for your contribution!

File: src/main/resources/view/MainWindowNew.fxml
Comment (stanleytangzh) [inline src/main/resources/view/MainWindowNew.fxml:1]: I like how the main window is displayed

**Pull request merged**